_lifeofinternatpuppet/post_3.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: "Week 3"
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 3
+---
+
+I had a very busy and productive week 3 at [Puppet](https://puppet.com/). 
+I started off my week 3 with the IDEA Working Group, where we discussed what to do with the upcoming events such as [Women Who Code](https://www.womenwhocode.com/)  taking opinions of existing interns of how they applied to Puppet for their placement year and if there was anything that can be improved.
+On the same day, I started to work on [Learn Ruby The Hard Way](https://learnrubythehardway.org/book/) 
+During the week, I continued to work on this tutorial when I was free and so far made great progress. Later on in the week, I had some very interesting and enjoyable sessions of training. 
+I had two sessions on [GitHub](https://github.com/) training which was a high-level introduction, covering things like:-
+
+**Part I:**
+
+-   What is SCM? Namespaces: puppetlabs, forks, etc
+-   Cloning a repo, pulling
+-   Creating a branch
+-   Adding, Commit and Removing etc
+-   How to create good commits (tagging with JIRA IDs, atomic commits, etc)
+-   Resolving conflicts (quick overview: working on a code repo with multiple collaborators / contributors)
+-   Pushing branches
+
+**Part II:**
+
+-   Push/pull destinations (e.g. upstream, origin, forks, etc)
+-   Creating a PR
+-   Code review basics (+ labelling)
+-   Rebasing Getting out of trouble (removing commits, rewriting commits, resetting branches)
+-   Force pushing and how to cautiously use it!
+-   Github issues and how to link to them in commits/PRs
+
+During this session, I worked with [Ciaran](https://github.com/sanfrancrisko) and  [Daiana](https://github.com/daianamezdrea).
+
+I have also started my training on [PDKSync and Gem Testing](https://github.com/puppetlabs/pdksync) with [Sheena](https://github.com/sheenaajay). 
+Check out the blogs on [PDKSync](https://puppet.com/blog/walkthrough-pdksync/). 
+The session will be continued on Monday to complete the remaining training. 
+In this session, the following topics were covered:-
+
+**Part I:**
```

Comment:
Should this be `Part III`?

Reply:
Hi, that's correct, sorry just a typo error, will fix it now